browser/brave_content_browser_client.cc

```diff
@@ -109,6 +110,10 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/components/brave_wallet/browser/brave_wallet_service.h"
 #endif
 
+#if BUILDFLAG(PRIVATE_CHANNEL_ENABLED)
+#include "brave/components/private_channel/client_private_channel.h"
```

Comment:
why are we adding a header here if no code has been added to use it?

Reply:
These are artifact from a previous version of the PR, good catch! Removing it.

- [x] remove unnecessary imports